pkg/porter/lifecycle.go

```diff
@@ -92,6 +99,47 @@ func (o *BundleExecutionOptions) GetParameters() map[string]interface{} {
 	return o.finalParams
 }
 
+// Sets the final resolved set of host volumes to be made availabe to the bundle
+func (o *BundleExecutionOptions) GetHostVolumeMounts() []cnabprovider.HostVolumeMountSpec {
```

Comment:
Will this handle spaces in the windows directory structure?

Reply:
Yes it does work. I have now added a test for the same. Thanks